.github/workflows/nix-check.yml

```
@@ -5,185 +5,173 @@ on:
+            - name: Checkout repository
+              uses: actions/checkout@v4
+
+            - name: Install Nix
+              uses: cachix/install-nix-action@v31
+              with:
+                  nix_path: nixpkgs=channel:nixos-unstable
+
+            - name: Run pre-commit checks
+              run: nix build .#checks.x86_64-linux.pre-commit --print-build-logs
+
+    # Flake metadata check
```

actionlint (1.7.10)

could not parse as YAML: did not find expected key

(syntax-check)